pkg/api/v1/types.go

```diff
@@ -2412,6 +2412,12 @@ type PodSpec struct {
 	// +patchMergeKey=IP
 	// +patchStrategy=merge
 	HostAliases []HostAlias `json:"hostMappings,omitempty" patchStrategy:"merge" patchMergeKey:"IP" protobuf:"bytes,23,rep,name=hostMappings"`
+	// If specified, indicates the pod's priority. "system" is a special keyword
+	// which indicates the highest priority. Any other name must be defined in
+	// Admission Controller config or pod will be rejected.
```

Comment:
Add a TODO to update this with the name of the admission controller once you write it.
BTW do you think you can do the admission controller for 1.7? It shouldn't be too hard if you mostly copy one of the existing admission controllers. I don't think any of the existing admission controllers are configured by a ConfigMap, though.